index.js

```diff
@@ -0,0 +1,11 @@
+//В try catch конструкцию завернуть код: console.log(a), let a = 3. И вывести ошибку – ‘let перед использованием нужно объявить’. При выполнении 1/0  выводить ошибку 'на ноль делить нельзя'
+
+try {
```

Comment:
You have two different tasks, the first is to catch an error for calling a variable before you declare it.
And the second task cannot be divided by zero to generate an error.
You should have 2 solutions, not just one. Change it, please.